src/app/modules/group/components/pending-request/pending-request.component.ts

```diff
@@ -164,7 +165,7 @@ export class PendingRequestComponent implements OnInit, OnChanges {
   }
 
   onAcceptOrReject(action: Action) {
-    if (this.selection.length === 0 || this.ongoingActivity !== Activity.None) {
+    if (this.selection.length === 0 || this.ongoingActivity !== Activity.None || !this.groupId) {
```

Comment:
But so you disable any action if there are no group? So it does not work. Wasn't it a working feature?

Reply:
Fixed